lib/foodcritic/rules/fc078.rb

```diff
@@ -0,0 +1,135 @@
+rule "FC078", "Ensure cookbook shared under an OSI-approved open source license" do
+  tags %w{supermarket}
```

Comment:
Now that I've experimented with `foodcritic` and `-t ~tag`, I see that the `~` wins regardless of what other tags are enabled. So, I suppose `opensource` might be better so that it can be the only tag on a rule that is only related to open source.

For example, `foodcritic -t metadata -t ~supermarket` turned off _any_ `metadata` rule that also had `supermarket` on it. 😞